### ISSUE TITLE
Tailwind CSS

### ISSUE THREAD

**Issue by lixiaoyan** [RightCapitalHQ/frontend-style-guide#274]
**ESLint:**
https://github.com/schoero/eslint-plugin-better-tailwindcss
We may enable all (not only recommended) rules for Tailwind CSS projects.
TBD: determine whether or not to enable the `no-unregistered-classes` rule. (may cause many false positives)

**Prettier:**
https://github.com/tailwindlabs/prettier-plugin-tailwindcss
The sorting function is already included in the ESLint plugin, therefore we don't need the official Tailwind CSS plugin.

---

```js
    plugins: {
      'better-tailwindcss': eslintPluginBetterTailwindcss,
    },
    rules: {
      // Tailwind CSS - stylistic
      'better-tailwindcss/enforce-consistent-class-order': 'error',
      'better-tailwindcss/enforce-consistent-important-position': 'error',
      'better-tailwindcss/enforce-consistent-line-wrapping': [
        'error',
        {
          // To avoid issues with the prettier formatter, we set the print
          // width to a value greater than the prettier's print width.
          printWidth: 120,
        },
      ],
      'better-tailwindcss/enforce-consistent-variable-syntax': 'error',
      'better-tailwindcss/enforce-shorthand-classes': 'error',
      'better-tailwindcss/no-deprecated-classes': 'error',
      'better-tailwindcss/no-duplicate-classes': 'error',
      'better-tailwindcss/no-unnecessary-whitespace': 'error',

      // Tailwind CSS - correctness
      'better-tailwindcss/no-conflicting-classes': 'error',
      'better-tailwindcss/no-restricted-classes': 'error',
      'better-tailwindcss/no-unregistered-classes': 'error',
    },
```
